src/routes/search.js

```diff
@@ -44,6 +44,11 @@ class SearchRoute extends Component {
     // cache queries so we can restore them with the search type buttons
     this.queries = {};
 
+    // cache indicators if a search or a filter has changed
+    // these impact the display of search panel on a small screen
```

Comment:
Since these only impact the _display_ of the search panel, could we move this logic into there somehow? This way, our routes are only concerned with data, and our components can handle their own display logic.

Reply:
Thanks @wwilsman - -Makes sense -- but just trying to understand where I could get context within the search-paneset to detect changes from a previous search to the current search. (for example, understanding current search has new search term or just new filters). The search route seemed logical for this, but I'll see what I can do in the paneset itself

Reply:
Great refactor 👍 makes a lot more sense to me to have that logic in the `SearchPaneset` component instead of the route.